### DISCUSSION TITLE
PDF 摘录时怎么能不导出用笔做的勾画呢？"Unsupported Type: ink"

### DISCUSSION THREAD

**Discussion by KKKZOZ** [aidenlx/obsidian-zotero#227] [Help]
![image](https://github.com/PKM-er/obsidian-zotlit/assets/70520782/f9f54c60-1977-407b-b548-03cd22c5a604)

就这种的话，模板要怎么写才能把勾画忽略掉呢

**Comment by aidenlx** [comment]
v2 里每条标注都带 `zt.type`（`highlight` / `note` / `image` / `ink` / `underline` / `text`），在模板里跳过 `ink` 即可：

```liquid
{% for annotation in zt.annotations %}
{% unless annotation.type == "ink" %}{% render "annotation" with annotation as zt %}{% endunless %}
{% endfor %}
```

不再有 `Unsupported Type` 这类报错 —— 各种类型都有对应的字段，要不要渲染由模板自己决定。

v2 beta 现已发布：https://zotlit.aidenlx.site/docs/install-zotlit
迁移指南：https://zotlit.aidenlx.site/docs/how-to/migrate-from-v1
